src/button.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { MouseEvent } from "react";
+import "./button.css";
+
+// HINT: Use this in ButtonProps to determine how the button is styled
+export enum ButtonType {
+  "button-green",
+  "button-white",
+  "button-clear",
+  "button-clearAlt",
+}
+
+// HINT: Use this in ButtonProps to determine how the button text is styled
+export enum ButtonTextType {
+  "button-text-white",
+  "button-text-green",
+  "button-text-grey",
+}
+
+interface ButtonProps {
+  text: string;
+  buttonType: string;
```

Comment:
Use `keyof typeof ButtonType` and `keyof typeof ButtonTextType` instead (:

Reply:
The reasoning behind it is that right now you're not harnessing those TypeScript superpowers to the full extent (; That is, by using `keyof typeof ButtonType` instead, we're able to force the "buttonType" to take in one of the values in the ButtonType enumeration!

Reply:
Oh, ok I see! I'll have to try utilizing that in the future to fully understand it, but thank you! That helps me understand it a lot better :DD